assets/js/app/Panel.js

```diff
@@ -66,50 +49,107 @@ LocusZoom.Panel = function() {
     this.getBaseId = function(){
         return this.parent.id + "." + this.id;
     };
+
+    // Initialize the layout (should this happen in initialize()?)
+    this.initializeLayout();
     
     return this;
     
 };
 
+LocusZoom.Panel.prototype.initializeLayout = function(){
+
+    // Set panel dimensions, origin, and margin or fall back to default values
+    this.layout.width      = this.layout.width      || 0;
```

Comment:
Well, hrm. So I looked at this and it's actually ambiguous how to pull these values form the default layout since the default layout has two panels with potentially distinct values. Should we just refer to one of the two default panels (positions/genes) as the "primary" and pull in default values from there? The pattern here with all 0s (or 1s for the proportional dimensions) are mainly just to make sure those values are defined and are integers.